lib/easy.c

```diff
@@ -775,6 +800,9 @@ static CURLcode dupset(struct Curl_easy *dst, struct Curl_easy *src)
   /* duplicate all strings */
   for(i = (enum dupstring)0; i< STRING_LASTZEROTERMINATED; i++) {
     result = Curl_setstropt(&dst->set.str[i], src->set.str[i]);
+    /* Curl_setstropt return CURLE_BAD_FUNCTION_ARGUMENT with blob */
+    if(result == CURLE_BAD_FUNCTION_ARGUMENT)
```

Comment:
I don't think it should call `Curl_setstropt` for the blobs at all since they're not strings!

Reply:
see comment on Curl_setblobopt below